ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -158,6 +213,14 @@ func (s *indexerStreamingService) publishTxn(ctx context.Context, req abci.Reque
 					continue
 				}
 				eventType := clonedEvent.Type
+				if eventType == gammtypes.TypeEvtTokenSwapped {
+					// Set the spot price for the token swapped event in the event's attributes map
+					err := s.setSpotPrice(ctx, clonedEvent)
+					if err != nil {
+						s.logger.Error("Error setting spot price", "error", err)
+						continue
+					}
+				}
```

Comment:
Just confirming - is the price only relevant for token-swapped event and not others? Does DexScreener not expect priceNative to be returned for other events?

Reply:
Thanks.
Correct. PriceNative is only for swap event.